tests/trainer/test_dataloaders.py

```diff
@@ -1032,23 +1032,77 @@ def test_dataloaders_load_only_once_no_sanity_check(tmpdir):
 @mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
 def test_dataloaders_load_every_epoch(tmpdir):
 
-    model = EvalModelTemplate()
+    model = BoringModel()
 
-    # logger file to get meta
     trainer = Trainer(
         default_root_dir=tmpdir,
         limit_train_batches=0.3,
         limit_val_batches=0.3,
-        reload_dataloaders_every_epoch=True,
+        reload_dataloaders_every_n_epochs=True,
         max_epochs=3,
     )
-    result = trainer.fit(model)
+    trainer.fit(model)
+    trainer.test()
 
+    # verify the sequence
+    calls = trainer.dev_debugger.dataloader_sequence_calls
+    expected_sequence = [
+        'val_dataloader',
+        'train_dataloader',
+        'val_dataloader',
+        'train_dataloader',
+        'val_dataloader',
+        'train_dataloader',
+        'val_dataloader',
+        'test_dataloader'
+    ]
+    for call, expected in zip(calls, expected_sequence):
+        assert call['name'] == expected
+
+
+@mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
+def test_dataloaders_load_every_n_epochs(tmpdir):
```

Comment:
can you merge this test with the previous one and use:

`@pytest.mark.parametrize("n", [1, 2])`